robocorp-code/src/robocorp_code/rcc.py

```diff
@@ -1046,6 +1046,7 @@ def return_failure(msg: Optional[str]) -> ActionResult[IRobotYamlEnvInfo]:
             str(conda_yaml_path),
         ]
         args.append("--json")
+        args.append("--no-retry-build")
```

Comment:
Can you also add `--no-pyc-management` here? It's something that should've been done quite a while ago too...